src/api/spec/models/project_spec.rb

```diff
@@ -768,4 +768,11 @@ def reset_project_in_backend
 
     it { expect(project.project_state).not_to be_nil }
   end
+
+  describe 'sphinx' do
```

Comment:
We have `RSpec.describe FullTextSearch`...

Reply:
I was looking for exactly that :slightly_smiling_face: